SA3/qa_web/tests/test_views.py

```diff
@@ -99,6 +126,10 @@ def test_asking_questions(self):
         self.assertEqual(post_questions_count, 1)
         self.assertEqual(tags[0], form_data['tag'])
 
+        form_data= {'wrong' : 'form'}
+        response = self.client.post('/questions/', data=form_data)
+        self.assertTemplateUsed(response, 'qa_web/questionspage.html')
```

Comment:
Do you think maybe we should add an assert here to indicate that actually the form is incorrect and nothing created with this form?

Reply:
Yes, the logic you described is correct here. I just asserted in the bigger picture, more in terms of end result. I can definitely try to refactor in another issue to be more specific.